contracts/bridges/L2BridgeExecutor.sol

```diff
@@ -1,20 +1,35 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity 0.8.10;
 
+import {IL2BridgeExecutor} from '../interfaces/IL2BridgeExecutor.sol';
 import {BridgeExecutorBase} from './BridgeExecutorBase.sol';
 
-abstract contract L2BridgeExecutor is BridgeExecutorBase {
+/**
+ * @title L2BridgeExecutor
+ * @author Aave
+ * @notice Abstract contract that implements bridge executor functionality for L2
+ * @dev It does not implement the `onlyEthereumGovernanceExecutor` modifier. This should instead be done in the inheriting
+ * contract with proper configuration and adjustments depending of the L2
+ */
+abstract contract L2BridgeExecutor is BridgeExecutorBase, IL2BridgeExecutor {
+  // Address of the Ethereum Governance Executor, able to queue actions sets
   address internal _ethereumGovernanceExecutor;
 
-  event EthereumGovernanceExecutorUpdate(
-    address previousEthereumGovernanceExecutor,
-    address newEthereumGovernanceExecutor
-  );
-
-  error UnauthorizedEthereumExecutor();
-
+  /**
+   * @dev Only the Ethereum Governance Executor can call functions marked by this modifier.
```

Comment:
```suggestion
   * @dev Only the Ethereum Governance Executor should be able to call functions marked by this modifier.
```

This and the other comment change I requested are just specifying that this is the intended behavior, but it isn't actually enforced here.

Reply:
Yes, good point